streams/src/main/java/org/apache/kafka/streams/state/internals/MergedSortedCacheWindowStoreIterator.java

```diff
@@ -43,7 +45,7 @@
 
     @Override
     Long deserializeCacheKey(final Bytes cacheKey) {
-        final byte[] binaryKey = bytesFromCacheKey(cacheKey);
+        final ByteBuffer binaryKey = bytesFromCacheKey(cacheKey);
         return WindowKeySchema.extractStoreTimestamp(binaryKey);
```

Comment:
Another example that we avoided the unnecessary array copy by returning sliced byte buffer via `bytesFromCacheKey`